docs/installation.md

```diff
@@ -136,134 +212,183 @@ using namespace jluna;
 int main()
 {
     State::initialize();
-    Base["println"]("hello julia");
+    Base["println"]("Your project is setup and working!");
 }
 ```
 
-then safe and close the file. 
+Then safe and close the file.
+
+### Our CMakeLists.txt
 
-To compile our project, we again use cmake. We first create `CMakeLists.txt`:
+Just like `jluna`, we will be building our own applicatoin using cmake. We create a `CMakeLists.txt` in `~/Desktop/MyProject`:
 
 ```bash
-# in ~/my_project/
+# in ~/Deskopt/MyProject
 gedit CMakeLists.txt
 ```
+And replace it's content with [this template `CMakeLists.txt`](../install/resources/CMakeLists.txt).
 
-And replace its contents with the following:
-
-```cmake
-cmake_minimum_required(VERSION 3.16)
+This `CMakeLists.txt` finds both Julia and `jluna` for us, then links them with our executable, and adds the include directory for `julia.h` and `jluna.hpp` to our paths. Note that this `CMakeLists.txt` depends on our project layout being exactly as detailed here, if we move `MyProject`, we may need to recompile `jluna` to updated any location-dependent code.
 
-# name of our project
-project(MyProject)
+### FindJulia.cmake
 
-# julia
-if (NOT DEFINED ENV{JULIA_PATH})
-    message(WARNING "JULIA_PATH was not set correctly. Consider re-reading the jluna installation tutorial at https://github.com/Clemapfel/jluna/blob/master/docs/installation.md#setting-julia_path to fix this issue")
-endif()
+To find julia, we need a `FindJulia.cmake`, which is provided by `jluna`. First, we create the corresponding file in `MyProject/cmake/find`:
```

Comment:
The only thing that the user needs is Julia itself, however that is provided. As I explained in my PR, because upstream Julia lacks a CMake package, consumers have to DIY the build tooling support. The fact that this project uses a find module is really only an implementation detail.